tests/api.c

```diff
@@ -14731,6 +14736,143 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
 
+/* Testing wc_SignatureGetSize() */
+static int test_wc_SignatureGetSize(void) {
+
+    int ret = 0; 
+    enum wc_SignatureType sig_type;
+    word32 key_len;
+    ecc_key ecc; 
+    RsaKey rsa_key;
+    byte* tmp = NULL;
+    size_t bytes;
+
+    /* Initialize ECC Key */
+    const char* qx = "fa2737fb93488d19caef11ae7faf6b7f4bcd67b286e3fc54e8a65c2b74aeccb0";
+    const char* qy = "d4ccd6dae698208aa8c3a6f39e45510d03be09b2f124bfc067856c324f9b4d09";
+    const char* d = "be34baa8d040a3b991f9075b56ba292f755b90e4b6dc10dad36715c33cfdac25";
+    
+    ret = wc_ecc_init(&ecc);
+    if (ret != 0) {
+        ret = WOLFSSL_FATAL_ERROR;
+        return ret;
```

Comment:
"ret" should be propagated down to the end of the function in error cases instead of being returned directly so that we can still catch the last "printf(resultFmt, ret == 0 ? passed : failed);" call.

This comment applies to other places where this occurs as well.